src/glados/settings.py

```diff
@@ -84,6 +84,16 @@ class RunEnvs(object):
     print("SSSearch results dir ({}) didn't exist, I will create it".format(SSSEARCH_RESULTS_DIR))
     os.mkdir(SSSEARCH_RESULTS_DIR)
 
+PROPERTIES_CONFIG_OVERRIDE_FILE = run_config.get('properties_config_dir')
+if PROPERTIES_CONFIG_OVERRIDE_FILE is None:
+    PROPERTIES_CONFIG_OVERRIDE_FILE = os.path.join(GLADOS_ROOT, 'es_properties_configuration/config/override.yml')
+print('PROPERTIES_CONFIG_OVERRIDE_FILE: ', PROPERTIES_CONFIG_OVERRIDE_FILE)
```

Comment:
here it is! Anyway, I also have some that need to be changed

Reply:
As discussed, this is not in the scope of the current task.